### DISCUSSION TITLE
Failed to create resource Certificate not issued in time

### DISCUSSION THREAD

**Discussion by sujitplus** [mozilla/hubs#4017] [Hubs Cloud Questions] [answered]
Hello,

I am unable to host Hubs Cloud on AWS. I have increased AWS Certificate Manager (ACM) to 100 but always getting error "Failed to create resource Certificate not issued in time"

I have tried deploying several time.

I tied with 2 domain purchased on AWS Route 53, not able to deploy.

Tried with 3 domain, one sub-domain and two domain purchased on Route 53.

I tried troubleshooting and narrow down the code that generate the error message.

event:
------
2021-03-05 10:52:40     Custom::DNSCertificate  InternalZoneSSLCert     arn:aws:acm:us-west-2:536782125463:certificate/db293024-96a2-453e-9cca-bdddbc2ca4f8  CREATE_FAILED       Failed to create resource Certificate not issued in time

From your template,
#code

def h():
     if e.get(b,K):raise U('Certificate not issued in time')
    e[b]=M;C(e);J('lambda').invoke(FunctionName=c.invoked_function_arn,InvocationType='Event',Payload=T(e))

Please help!!

thanks,
Sujit

**Comment by sujitplus** [comment, marked as answer]
Problem got solved, the issue is with DNS

**Reply by johnshaughnessy** [reply]
@sujitplus Do you mind expanding on what the issue with DNS was so that if another user encounters this problem we might be able to help them?

**Reply by angelcabrera-nr** [reply]
In my case, in my domain provider I do not have the AWS name servers configured correctly, therefore route53 / certificate manager, I could not access my domain
